src/index.ts

```diff
@@ -186,6 +186,9 @@ async function fetchData() {
   const isIncludeBacklinks = await joplin.settings.value(
```

Comment:
Can we call is `shouldIncludeBacklinks`?

Reply:
See https://github.com/treymo/joplin-link-graph/pull/55#discussion_r811449353